test/integration/wiki_creation_test.rb

```diff
@@ -36,7 +36,7 @@ class WikiCreationTest < ActionDispatch::IntegrationTest
 
     assert_nil node
     assert_equal '/wiki/create', path
-    assert_select 'h2', '1 error prohibited this revision from being saved'
```

Comment:
When I am adding this the tests are failing. I can't find out why this is happening. So, I have deleted the test in red and all tests are passing. On the localhost, it is displaying a red rectangle without text if I try to submit wiki page with no body or no title.

Reply:
Hmm, you mean it's showing a red rectangle without text and your PR fixes that, or that's a remaining problem the PR doesn't get to? 

Perhaps we can open a new issue with a sub-issue as described here.

Reply:
Could you post a screenshot of what you see from the code in this PR? Thank you! I just want to be careful with this one.

Reply:
Yeah, we can open an issue for the red rectangle.